animalfacts.py

```diff
@@ -1306,6 +1307,19 @@ def animalfactsbot(reddit):
     'Lobsters keep growing forever. They do not get weaker or lose their ability to reproduce, and will keep on molting and growing.',
     )
 
+MANTIS_SHRIMP_FACTS = (
+    'Mantis Shrimps can see ultraviolet and polarized light.',
+    'Mantis Shrimp have trinocular vision, meaning they can see using three parts of the same eye.',
+    'Mantis Shrimp can be categorized into "Spearers" and "Smashers", referencing tactics used to kill their prey.',
+    'A Mantis Shrimp can punch at a speed of 10 meters per second, equivalent to a .22 bullet.',
+    'Mantis Shrimp are known to break glass aquariums with their punches.',
+    'Mantis Shrimp are not actually shrimp, they are instead stomatopods, a distant relative.',
+    'A punch of a Mantis Shrimp momentarily causes the temperature of the surrounding water to reach the temperature of the Sun.',
+    'Mantis Shrimp use their ability to see polarized light to commmunicate with other mantis shrimp in a way that is invisible to predators.',
+    'Mantis Shrimp can make a low growling sounds and often grunt at dawn and dusk.',
+    'Stomatopods, like Mantis Shrimp, are older than dinosaurs.'
+    )
+
```

Comment:
Mantis shrimps have the most complex eyes in the animal kingdom that can see ultraviolet and polarized light. They have trinocular vision which means that they can see one objects using one of the three different parts of eye.